src/helpers/helpers/plot.py

```diff
@@ -56,4 +56,4 @@ def group_roc_curves(labels, scores, attr):
             "xaxis": {"title": "False Positive Rate"},
             "yaxis": {"title": "True Positive Rate"},
         },
-    )
+    )
```

Comment:
suspect no new line at the end of the file will fail flake8 if the tests were running. since this file is unchanged other than whitespace we can maybe revert it?